scripts/start/submit.R

```diff
@@ -106,13 +108,15 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
   
   # send prepare_and_run.R to cluster 
   cat("   Executing prepare_and_run.R for",cfg$results_folder,"\n")
-  if (grepl("^direct", cfg$slurmConfig)) {
-    log <- format(Sys.time(), paste0(cfg$title,"-%Y-%H-%M-%S-%OS3.log"))
-    system("Rscript prepare_and_run.R")
+  if (grepl("^direct", cfg$slurmConfig) || ! isSlurmAvailable()) {
+    exitCode <- system("Rscript prepare_and_run.R")
   } else {
-    system(paste0("sbatch --job-name=",cfg$title," --output=log.txt --mail-type=END --comment=REMIND --wrap=\"Rscript prepare_and_run.R \" ",cfg$slurmConfig))
+    exitCode <- system(paste0("sbatch --job-name=",cfg$title," --output=log.txt --mail-type=END --comment=REMIND --wrap=\"Rscript prepare_and_run.R \" ",cfg$slurmConfig))
```

Comment:
So this means in case we have multiple scenarios in the test/config files, they would not run in parallel, but one after the other?

Reply:
If we specify multiple scenarios in test config files and they specify `--wait` in the slurm config, then they are not run in parallel, that's correct. When we have more tests and want to run them in parallel, we'll need to figure something out.